service-runner/src/main/resources/profiles/default/test.env

```diff
@@ -4,10 +4,13 @@ sep.server.url=http://localhost:8080
 reference.server.url=http://localhost:8091
 observer.server.url=http://localhost:8083
 platform.server.url=http://localhost:8085
+custody.server.url=http://localhost:8086
 wallet.server.url=http://localhost:8092
 run_docker=true
 run_all_servers=true
 
 # Enable the test flows in the kotlin reference server
-sep24.enableTest=true
+app.enableTest=true
```

Comment:
We should make `app.enableTest=true`, app.custodyEnabled=false` and `app.rpcEnabled=false` as default values